code/modules/mob/living/simple_animal/eldritch_demons.dm

```diff
@@ -0,0 +1,272 @@
+/mob/living/simple_animal/hostile/eldritch
+	name = "Demon"
+	real_name = "Demon"
+	desc = ""
+	gender = NEUTER
+	mob_biotypes = NONE
+	speak_emote = list("screams")
+	response_help_continuous = "thinks better of touching"
+	response_help_simple = "think better of touching"
+	response_disarm_continuous = "flails at"
+	response_disarm_simple = "flail at"
+	response_harm_continuous = "reaps"
+	response_harm_simple = "tears"
+	speak_chance = 1
+	icon = 'icons/mob/eldritch_mobs.dmi'
+	speed = 0
+	a_intent = INTENT_HARM
+	stop_automated_movement = 1
+	AIStatus = AI_ON
+	attack_sound = 'sound/weapons/punch1.ogg'
+	see_in_dark = 7
+	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
+	damage_coeff = list(BRUTE = 1, BURN = 1, TOX = 0, CLONE = 0, STAMINA = 0, OXY = 0)
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	maxbodytemp = INFINITY
+	healable = 0
+	faction = list("e_cult")
```

Comment:
Legacy?

Reply:
i changed the name purely cosmetically, same way obessed is called creep in code